### PR TITLE
feat(app-service-configurable-rules): Replace Export services with app-service-configurable

### LINKED ISSUE #168: Replace Export Services with App Service Configurable (rules-engine profile)
App Service Configurable running the rules-engine profile is now the default way for messages to be passed to the Rules Engine. The Rules Engine properties have been modified so that the Rules Engine will receive message from  App Service Configurable.

App Service Configurable running the rules-engine profile needs to be added to all the nightly build compose files.

The Export Services (Client & Distro) will be commented out so that they may be used if someone prefers run them in parallel with the App Service Configurable running the rules-engine profile.

### PR DESCRIPTION
**Pull request by lenny-intel**
New default is for app-service-configurable running rules-engine profile to provide the event message to the rules engine.

closes #168

### CODE REVIEW

File: releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml [hunk 1]
```diff
@@ -229,63 +233,81 @@ services:
     depends_on:
       - metadata
 
-  export-client:
-    image: nexus3.edgexfoundry.org:10004/docker-export-client-go:1.1.0
+  app-service-rules:
+    image: nexus3.edgexfoundry.org:10004/docker-app-service-configurable:master
```
Comment (jim-wang-intel) [inline releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml]: Should this image have a version like `1.1.0`?

File: releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml [hunk 2]
```diff
@@ -229,63 +233,81 @@ services:
...
-      <<: *common-variables
-      EXPORT_DISTRO_CLIENT_HOST: export-client
-      EXPORT_DISTRO_DATA_HOST: edgex-core-data
-      EXPORT_DISTRO_CONSUL_HOST: dgex-config-seed
-      EXPORT_DISTRO_MQTTS_CERT_FILE: none
-      EXPORT_DISTRO_MQTTS_KEY_FILE: none
-
   rulesengine:
-    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:1.0.0
+    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:master
```
Comment (jim-wang-intel) [inline releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml]: Should this image have a version like `1.1.0`?

File: releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty.yml [hunk 3]
```diff
@@ -220,60 +224,81 @@ services:
       - metadata
       - redis
 
-  export-client:
-    image: nexus3.edgexfoundry.org:10004/docker-export-client-go:1.1.0
+  app-service-rules:
+    image: nexus3.edgexfoundry.org:10004/docker-app-service-configurable:master
```
Comment (jim-wang-intel) [inline releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty.yml]: Ditto

File: releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty.yml [hunk 4]
```diff
@@ -220,60 +224,81 @@ services:
...
-      <<: *common-variables
-      EXPORT_DISTRO_CLIENT_HOST: export-client
-      EXPORT_DISTRO_DATA_HOST: edgex-core-data
-      EXPORT_DISTRO_CONSUL_HOST: edgex-config-seed
-      EXPORT_DISTRO_MQTTS_CERT_FILE: none
-      EXPORT_DISTRO_MQTTS_KEY_FILE: none
-
   rulesengine:
-    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:1.0.0
+    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:master
```
Comment (jim-wang-intel) [inline releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty.yml]: Ditto

File: releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml (hunk 1 above)
Comment (lenny-intel) [reply releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml]: `master` for the images created from the new CI Pipelines. Eventually, all will be changed to `master` for nightly. When Fuji versions are created from nightly compose files, all the tags will get changed to the appropriate version number.

File: releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml (hunk 2 above)
Comment (lenny-intel) [reply releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml]: See previous response

File: releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml (hunk 1 above)
Comment (jim-wang-intel) [reply releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml]: Got cha

**Comment by lranjbar** [comment]
@lenny-intel Please do a rebase - this repo has been moving somewhat fast

**Pull request merged**